frontend/app/error.vue

```diff
@@ -5,8 +5,25 @@ import { setFavicon } from '~/utils/setFavicon'
 const props = defineProps({
   error: Object as () => NuxtError,
 })
-
+const showTimer = ref(false)
+const redirectPage = ref('/tasks')
 useOgImage()
+const { data } = await useApi('/event/info')
+
+const futureDate = computed(() => new Date(data.value?.start_date ?? Date.now()))
+function redirectToPage() {
+  navigateTo(redirectPage.value)
+}
+
+watchEffect(() => {
```

Comment:
Xddddd, since when we declare variables outside and then use watchEffect to update them? Why not `computed`, like we already use it for the `errorMessage`?